ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButtonGroupView: UIView {
+    let buttons: [CommandBarButton]
+
+    init(buttons: [CommandBarButton]) {
+        self.buttons = buttons
+
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = false
+
+        clipsToBounds = true
+        layer.cornerRadius = CommandBarButtonGroupView.cornerRadius
+        if #available(iOS 13, *) {
+            layer.cornerCurve = .continuous
+        }
+
+        configureHierarchy()
+        applyInsets()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: buttons)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.spacing = CommandBarButtonGroupView.buttonPadding
+
+        return stackView
+    }()
+}
+
+private extension CommandBarButtonGroupView {
+    func configureHierarchy() {
+        addSubview(stackView)
+        NSLayoutConstraint.activate([
```

Comment:
you can avoid all the complex layout logic by subclassing this from `UIStackview` instead of `UIView`. 
Everything including padding and adding inset is supported by stackview.

Reply:
If `CommandBarButtonGroupView` is exposed as `UIStackView` subclass, callers may have the chance of calling `UIStackView` API (e.g. `addArrangedSubview`), which is not expected by the designer of `CommandBarButtonGroupView` (all the `arrangedSubviews` should be managed by `CommandBarButtonGroupView` internally).

Furthermore,  If we later want to make `CommandBarButtonGroupView` inherit from another `UIStackView` substitution, there will be an API breaking change (users may already depend on the inherited `UIStackView` interface). So it seems appropriate to me that using a wrapper view to hide the internal implementation with `UIStackView`.

Reply:
Before iOS 14, `UIStackView` is non rendering view, making it unable to have rounded corners.

Reply:
@zhuorantan Thanks for brining this to my attention  if we can document the need for additional wrapper as a requirement for iOS 13 then we can remove them later. 

I understand the need for not exposing `UIStackView` api's. I could argue the same thing that exposing `UIView` could have unexpected consequences in other scenarios. A final subclass of `UIStackView` that suppress the unintended consequences could be used a drop in replacement instead. 

My primary concerns by using too many wrapper views are accessibility bugs, boilerplate code and forward compatibility issues with UIKit. This comment is **not** a blocking issue.